.agent/workflows/manage-auction.md

```
@@ -0,0 +1,70 @@
+
+### Step 5: Update the Database
+1. Open `data/auctions.json` using the replace or multi-replace tool.
+2. Modify the target auction object:
+   - Set `"status"` to `"completed"`.
+   - Add `"winningBid"` (e.g., the selected bid amount).
+   - Add `"acquiredBy"` (e.g., the winner's first name / company or "Anonymous Creator").
+   - If a new round is planned, scaffold the next slot as `"upcoming"` or `"active"` with new dates.
+
+### Step 6: Deploy & Sync
+1. Run local verification gates:
+   ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)